clients/filesystem-fuse/check_rust_env.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+if ! command -v cargo &> /dev/null; then
```

Comment:
rename the file to `install_rust_env.sh`, and place it in `dev` directory?

Reply:
Added.  
I think the `install_rust_env.sh` file does not need to be moved to the dev directory. We need to extend the script to check and install other tools for `gvfs-fuse`